map/src/components/header.tsx

```diff
@@ -33,24 +39,16 @@ const Header = (props: Props) => {
             <img src="/logo-compat.svg" alt="Reach4Help Logo" />
           </div>
           <div className="info">
-            <h1>COVID-19 Mutual Aid Map - Reach4Help</h1>
-            <p>
-              To help people find and join mutual aid efforts where they live,
-              offer inspiration to start their own, and/or simply lift spirits,
-              below is a growing list of mutual aid pandemic disaster care
-              projects. Note: Each project is autonomous and self-organized;
-              many use public spreadsheets to share information, so be careful
-              when entering private information that you don&apos;t want to be
-              public.
-            </p>
+            <h1>{t(s => s.title)} - Reach4Help</h1>
+            <p>{t(s => s.info)}</p>
             <p className="muted">
-              This map is part of&nbsp;
-              <a href="https://reach4help.org">reach4help.org</a>, a
-              volunteer-run project. It is open source and can be&nbsp;
+              {t(s => s.source1)}&nbsp;
+              <a href="https://reach4help.org">reach4help.org</a>
+              {t(s => s.source2)}&nbsp;
```

Comment:
probably not necessary for this PR, but we could do with using placeholders, or allowing template functions with HTML escaping instead of having to split translation around html components like this.

Placeholders could be achievable in this PR though.